optimum/intel/openvino/modeling_visual_language.py

```diff
@@ -617,11 +617,8 @@ def forward(
         position_ids=None,
         image_bound=None,
         tgt_sizes=None,
-        images=None,
         **kwargs,
     ):
-        if pixel_values is None and images is not None:
-            pixel_values = images
```

Comment:
and what is suggestion, let nanollava fails?

Reply:
I've fixed the current issue with nano-llava inside `_OVNanoLlavaForCausalLM.preprocess_inputs()`

Reply:
``` 
FAILED tests/openvino/test_modeling.py::OVModelForVisualCausalLMIntegrationTest::test_compare_to_transformers_2_nanollava - TypeError: forward() got an unexpected keyword argument 'pixel_values'
```
looks like also need to update transformers_model input in test as it uses images as input name instead of pixel_values

Reply:
I updated both preprocess_input and test in https://github.com/huggingface/optimum-intel/pull/967

Reply:
Thanks! I reverted these changes in this PR